dotnetv3/Kinesis/README.md

```diff
@@ -1,37 +1,36 @@
-# Amazon Kinesis .NET code examples
+# Amazon Kinesis code examples for .NET
 
-## Purpose
+## Overview
 
-This folder contains examples that show how to use the AWS SDK for .NET 3.x to
-get started working with Amazon Kinesis features such as streams and consumers.
+The examples in this section show how to use the AWS SDK for .NET with Amazon Kinesis features such as streams and consumers.
 
-With Kinesis, you can ingest real-time data such as video, audio, application
-logs, website clickstreams, and IoT telemetry data for machine learning,
-analytics, and other applications. Kinesis enables you to process and
-analyze data as it arrives and respond instantly instead of having to wait
-until all your data is collected before the processing can begin.
+Amazon Kinesis makes it easy to collect, process, and analyze real-time, streaming data so you can get timely insights and react quickly to new information. Amazon Kinesis offers key capabilities to cost-effectively process streaming data at any scale, along with the flexibility to choose the tools that best suit the requirements of your application.
```

Comment:
Amazon Kinesis makes it easy to collect, process, and analyze video and data streams in real time.

Reply:
Fixed.